packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createProbot, Probot, ApplicationFunction, Options, Application } from 'probot';
+import { resolve } from 'probot/lib/resolver';
+import { Storage } from '@google-cloud/storage';
+import * as KMS from '@google-cloud/kms';
+import { readFileSync } from 'fs';
+import * as express from 'express';
+
+
+export class GCFBootstrapper {
+  probot?: Probot;
+
+  async loadProbot(appFn: ApplicationFunction): Promise<Probot> {
```

Comment:
how are you picturing applications will bootstrap, does it make sense to expose this as a bin, so that ultimately we just having something like:

```json
{
  "scripts": {
    "prod": "cf-probot ./build/my-bot.js"
  }
}
```

Reply:
I would imagine it would be something similar to the following:

``` typescript
import { GCFBootstrapper } from '../gcf-utils';
import * as appFn from './';

bootstrap = new GCFBootstrapper();
module.exports = bootstrap.gcf(appFn);
```

With that being the "main" that is loaded as a Cloud Function.

It is fairly boilerplate, but I think holds in convention with other Probot apps && will allow us to write our "core" Probot apps cross-functional if need be.

What do you think?

Reply:
this seems reasonable to me, I think we can probably just add a `bin/run.js` for our bots, which the boiler plate you shared, and then make `npm run` call this 👍